apps/charterafrica/src/components/OpportunityCardList/OpportunityCardList.js

```diff
@@ -0,0 +1,115 @@
+import Link from "@/commons-ui/next/Link";
+import { Typography, Box, Grid } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import React from "react";
+
+import OpportunityCard from "../OpportunityCard";
+
+import { neutral } from "@/charterafrica/colors";
+
+const OpportunityCardList = React.forwardRef(function OpportunityCardList(
+  props,
+  ref
+) {
+  const { title, grants } = props;
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
+  const isSmallDesktop = useMediaQuery(theme.breakpoints.only("md"));
+  return (
+    <Box ref={ref}>
+      <Grid
+        container
+        justifyContent="space-between"
+        sx={{
+          marginBottom: "40px",
+        }}
+      >
+        <Grid item>
+          <Typography
+            variant="h5SemiBold"
+            color={neutral[900]}
+            sx={{
+              textTransform: "capitalize",
+            }}
+          >
+            {title}
+          </Typography>
+        </Grid>
+        <Grid item>
+          <Link
+            href="/grants"
+            underline="always"
+            color={neutral[900]}
+            sx={{
+              variant: "p3SemiBold",
+              display: {
+                xs: "none",
+                md: "block",
+              },
+            }}
+          >
+            View All
+          </Link>
+        </Grid>
+      </Grid>
+      <Grid
```

Comment:
```suggestion
      <Grid
        item
```
Everything inside a `Grid container` must be a `Grid item` (a `Grid item` can also be a `Grid container` at the same time of course i.e. `Grid item xs={12} container ...` or you can make the container child of the item i.e. `Grid item xs={12}><Grid container ...`). Please check the docs again.